lib/artirix_data_models/cached_action_adaptor.rb

```diff
@@ -44,10 +44,10 @@ def enabled?
   private
 
   def get_cached_result
-    return nil unless enabled?
+    raise "Unexpected NilClass returned from cache - case 2" unless enabled?
```

Comment:
this will *ALWAYS* fail in development and INT. it needs to return `nil` if the cache is not enabled. returning `nil` in cache adaptor is "I do not have this cached".